packages/office-ui-fabric-react/src/components/Announced/Announced.types.ts

```diff
@@ -1,12 +1,11 @@
 import * as React from 'react';
-import { AnnouncedBase } from './Announced.base';
 import { IStyle } from '../../Styling';
 import { IStyleFunctionOrObject } from '../../Utilities';
 
 /**
  * {@docCategory Announced}
  */
-export interface IAnnouncedProps extends React.Props<AnnouncedBase>, React.HTMLAttributes<HTMLDivElement> {
+export interface IAnnouncedProps extends React.PropsWithChildren<React.HTMLAttributes<HTMLDivElement>> {
```

Comment:
Will this and the other changes be fully backwards compatible? Especially concerned about the React.Props changes.

Reply:
I'm with Elizabeth here, this looks like it could be a breaking change as it removes `key` and `ref` from typings. I think it would be ok as long as we can claim correctness for any typing breaks (i.e. `key` and `ref`, if used, have absolutely no effect and shouldn't be supplied anyways), but I also want to make sure we shouldn't look at using other types like `PropsWithRef` in each of these cases.

If and when we do introduce a change explicitly adding children to props, we should rationalize why. [The typings automatically add `children` in the component definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/267dc9f23ae19f38f414ee02f8d7ac738aabad60/types/react/index.d.ts#L518), making it available when children are used in JSX. I have a feeling any `WithChildren` types should be explicitly used in nearly all component typings or defer to React component types. I'm guessing it will be the latter. 

Overall, I think this change should be part of a larger component typing strategy suitable for component typing guidelines (ref usage, ref forwarding, children specification, key usage, etc.) and conformance testing for our next major release.

Reply:
I'll roll back on these changes.

Reply:
Reverted these changes.